package.json

```diff
@@ -7,7 +7,7 @@
   "dependencies": {
     "@dsnp/contracts": "^0.0.0-5c945f",
     "@dsnp/parquetjs": "^1.1.0",
-    "@dsnp/sdk": "2.0.0",
+    "@dsnp/sdk": "^0.0.0-6082ae",
```

Comment:
why roll back?

Reply:
This is actually moving forward. The method to subscribe to registry events is not yet in a release.